src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionCoordinatorTests.swift

```diff
@@ -46,7 +49,7 @@ class ExposureSubmissionCoordinatorTests: XCTestCase {
 		delegate: ExposureSubmissionCoordinatorDelegate) -> ExposureSubmissionCoordinator {
 
 		return ExposureSubmissionCoordinator(
-			parentNavigationController: parentNavigationController,
+			warnOthers: WarnOthers(store: self.store), parentNavigationController: parentNavigationController,
```

Comment:
Please add linebreak after added parameter and check in other places as well, I've seen this multipler times

Reply:
Done